spec/database_helper.rb

```diff
@@ -0,0 +1,36 @@
+require "sequel"
+
+class DatabaseHelper
+  USER = ENV.fetch("DATABASE_USER")
```

Comment:
Oh yes, I completely agree. `dotenv` / `.env` for tests is no good, can in fact be dangerous because sometimes (only sometimes) we populate our local machines with... let's says "non local values" to do stuff.